tools/kvrocks2redis/config.cc

```diff
@@ -41,11 +41,25 @@ Status Config::parseConfigFromString(std::string input) {
       return Status(Status::NotOK, "argument must be 'yes' or 'no'");
     }
     daemonize = (i == 1);
-  } else if (size == 2 && args[0] == "dir") {
-    dir = args[1];
-    db_dir = dir + "/db";
-    pidfile = dir + "/kvrocks2redis.pid";
-    next_seq_file_path = dir + "/last_next_seq.txt";
+  } else if (size == 2 && args[0] == "data-dir") {
```

Comment:
this would have a breaking change for the old version?

Reply:
I think it's a tool that won't be run by users for a long time, so making these changes doesn't affect users much, right?

Reply:
Okay, it's fine to me.